### PR TITLE
Update to v0.14.0-rc3

### PR DESCRIPTION
**Pull request by JordanMartinez**
Backlinking to purescript/purescript#3942

### CODE REVIEW

**Review by kl0tl — APPROVED** [review]

File: spago.dhall [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{-
+Welcome to a Spago project!
+You can edit this file as you like.
+-}
```
Comment (kl0tl) [inline spago.dhall]: Shall we remove this comment?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: CI
 
-on: push
+on: [push, pull_request]
```
Comment (thomashoneyman) [inline .github/workflows/ci.yml]:
```suggestion
on:
  push:
    branches: [master]
  pull_request:
    branches: [master]
```

File: spago.dhall (hunk 1 above)
Comment (thomashoneyman) [inline spago.dhall]:
```suggestion
```

File: spago.dhall [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+{ name = "web-cssom"
+, dependencies = [ "web-dom", "web-html", "psci-support" ]
+, packages = ./packages.dhall
+, sources = [ "src/**/*.purs", "test/**/*.purs" ]
```
Comment (thomashoneyman) [inline spago.dhall]:
```suggestion
, sources = [ "src/**/*.purs" ]
```

We can either trim the sources or add a dummy test.

**Review by thomashoneyman — CHANGES_REQUESTED** [review]

**Comment by thomashoneyman** [comment]
@JordanMartinez looks like this library was missed when updating them all to 0.14.0-rc3 with Bower.

**Comment by JordanMartinez** [comment]
@thomashoneyman Thanks. I've updated my PR to use bower rather than Spago

**Comment by JordanMartinez** [comment]
CI builds

**Pull request merged**